Documentation/Stage-1-Introduction/Day-1/Index.rst

```diff
@@ -54,10 +55,7 @@ Practical prerequisites
 
 Before you start this day, please have the following things ready:
 
-* Item 1
-* Item 2
-* Item 3
-
+* Access to the `official TYPO3 Documentation <https://docs.typo3.org/>`__
```

Comment:
The documentation is more like a learning resource for this day and added to a resource section. But if so, it should be more specific to what the student needs to learn.

I would add a sentence here that just refers to the practical prerequisites for stage 1. You can make a link to that section like this:

```
:ref:`Practical Prerequisites for Stage 1 <s1-practical-prerequisites>`
```